portfolio/src/main/webapp/template.html

```diff
@@ -68,73 +68,39 @@ <h2 class="heading">Latest Projects</h2>
                             <div class="item featured text-center">
                                 
                                 <div class="featured-image has-ribbon">
-                                    <a href="https://themes.3rdwavemedia.com/bootstrap-templates/startup/launch-bootstrap-4-template-for-saas-businesses/" target="_blank">
-                                    <img class="img-fluid project-image rounded shadow-sm" src="assets/images/projects/project-featured.jpg" alt="project name" />
+                                    <a href="#" target="_blank">
+                                    <img class="img-fluid project-image rounded shadow-sm" src="assets/images/projects/GradTracks.png" alt="GradTracks" />
                                     </a>
                                     <div class="ribbon">
                                         <div class="text">New</div>
                                     </div>
                                 </div>
                                 
-                                <h3 class="title mb-3"><a href="https://themes.3rdwavemedia.com/bootstrap-templates/startup/launch-bootstrap-4-template-for-saas-businesses/" target="_blank">GradTracks - The best way to visualize alumni paths and outcomes</a></h3>
+                                <h3 class="title mb-3"><a href="#" target="_blank">GradTracks - The best way to visualize alumni paths and outcomes</a></h3>
```

Comment:
Per the style guide, new lines for every block and indent every child element here and everywhere:
https://google.github.io/styleguide/htmlcssguide.html#HTML_Line-Wrapping